### PR TITLE
new records for BLorient535

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

File: new/LIT5868Hymn.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition">
+                <div type="textpart" subtype="incipit" xml:lang="am">
+                    <note>The following incipit is taken from <ref type="mss" corresp="BLorient535"/>.</note>
+                    <ab>መወድስ፡ ዘዘማርያም፡ የስንዴ፡ ዛላ፡ ወሪቅ፡ ድሜጥሮስ፡ ድሜጥሮስ፡ ለጋ፡ የዶግ፡ ድሜጥሮስ፡ ለምለምለም፡ 
+                        ስንደዶ፤ በክሮ፡ ሎሚ፡ ድሜጥሮስ፡ የመጌዶ፤</ab>
+                </div>
```
Comment (eu-genia) [inline new/LIT5868Hymn.xml]: isAbout relation could be helpful (PRS3499Demetriu?) but of course it is up to you to decide

Comment (DenisNosnitsin1970) [reply new/LIT5868Hymn.xml]: የዶግ፡ ድሜጥሮስ፡ ለምለምለም፡ - ??? If you did not mark it - I think "mawaddes" in the beginning is the type of the hymn ("mawaddes written by Za-Maryam"), and the language seems to be Amharic.

Comment (thea-m) [reply new/LIT5868Hymn.xml]: It is a bit to unclear for me to identify PRS3499Demetriu with certainty...

Comment (DenisNosnitsin1970) [reply new/LIT5868Hymn.xml]: Demetros is the protagonist of thie "mawaddes"/praise. I am not sure that we need here an ID for him, rather for Za-Maryam as the author.

**Pull request merged**